File: student_ahp_system_FE/lib/Models/risk_result.dart
class RiskResult {
  final int studentId;
  final double riskScore;
  final String riskLevel;
  final String calculatedDate;

  RiskResult({
    required this.studentId,
    required this.riskScore,
    required this.riskLevel,
    required this.calculatedDate,
  });

  factory RiskResult.fromJson(Map<String, dynamic> json) {
    return RiskResult(
      studentId: json['studentId'],
      riskScore: json['riskScore'].toDouble(),
      riskLevel: json['riskLevel'],
      calculatedDate: json['calculatedDate'],
    );
  }
}

File: AHP_system_BA/Models/RiskResult.cs
namespace DSSStudentRisk.Models;
public class RiskResult
{
    public int Id { get; set; }  

    public int StudentId { get; set; }

    public double RiskScore { get; set; }

    public string? RiskLevel { get; set; }

    public DateTime CalculatedDate { get; set; }
    public Student? Student { get; set; }
}